### PR TITLE
Validate the uploaded zipfile

### PR DESCRIPTION
**Pull request by sumandari**
This PR refers to #1252.
It provides validation during uploading the zipfile.

image

### CODE REVIEW

File: django_project/lesson/utilities.py [hunk 1]
```diff
@@ -60,3 +64,23 @@ def re_order_features(request, features):
...
+        zip = zipfile.ZipFile(file)
+    except:
+        raise ValidationError(_("Could not unzip file."))
+    for zname in zip.namelist():
+        if zname.find('..') != -1 or zname.find(os.path.sep) == 0:
+            raise ValidationError(
+                _('For security reasons, zip file cannot contain path '
+                  'informations')
+            )
+        for forbidden_dir in ['__MACOSX', '.git', '__pycache__']:
```
Comment (dimasciput) [inline django_project/lesson/utilities.py]: @sumandari it would be nice if we put this list in something like an ignore list file.

**Review by dimasciput — APPROVED** [review]
LGTM

**Comment by sumandari** [comment]
@dimasciput I updated with ignore list. Would please kindly re-review it. Thank you

**Comment by dimasciput** [comment]
LGTM!

**Pull request merged**